test/config_parser_SUITE.erl

```diff
@@ -1036,45 +1035,54 @@ auth_dummy(_Config) ->
 %% tests: outgoing_pools
 
 pool_type(_Config) ->
-    ?cfg(pool_config({http, global, default, [], []}),
+    ?cfg(pool_config(#{type => http,
```

Comment:
I think `pool_config` sets some defaults like `scope => global` and the default opts. Could we use these?

Reply:
Good idea, should be done like this now